analytics_data_api/v0/exceptions.py

```diff
@@ -0,0 +1,52 @@
+import abc
+
+
+class BaseError(Exception):
+    """
+    Base error.
+    """
+
+    __metaclass__ = abc.ABCMeta
```

Comment:
I'm preventing others from creating an instance of `BaseError` directly and use the other concrete classes defined in the file.  I believe without `abc` once could still create an instance of an `BaseError` otherwise.